edx_exams/apps/core/admin.py

```diff
@@ -61,7 +61,10 @@ class ExamAttemptAdmin(admin.ModelAdmin):
 
     def change_view(self, request, object_id, form_url='', extra_context=None):
         extra_context = extra_context or {}
-        extra_context['LTI_ROOT'] = get_lti_root()
+        # technically this should be be using get_lti_view_base() but the
```

Comment:
the customization on this django view can eventually be removed. It was only needed when we didn't have a UI to test with. I didn't want to add all that to this PR tho